ibm/conns/config.go

```diff
@@ -985,6 +991,14 @@ func (sess clientSession) SatelliteClientSession() (*kubernetesserviceapiv1.Kube
 	return sess.satelliteClient, sess.satelliteClientErr
 }
 
+// CIS Webhooks
+func (sess clientSession) CisWebhooksSession() (*ciswebhooksv1.WebhooksV1, error) {
+	if sess.cisWebhooksErr != nil {
+		return sess.cisWebhooksClient, sess.cisWebhooksErr
+	}
+	return sess.cisWebhooksClient.Clone(), nil
+}
+
```

Comment:
Add this `session.cisFiltersErr = errEmptyBluemixCredentials` [here](https://github.com/IBM-Cloud/terraform-provider-ibm/blob/f7527f348cdc36455133aeb648c2c3f7d331dc7d/ibm/conns/config.go#L1139)